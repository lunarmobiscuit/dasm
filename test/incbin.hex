:03C000000000003D
:00000001FF
