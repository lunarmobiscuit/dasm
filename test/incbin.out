unable to open incbinexample1.bin
unable to open incbinexample1.bin
unable to open incbinexample1.bin

Complete. (0)
